Makefile

```diff
@@ -4,7 +4,7 @@ export GOPROXY=https://proxy.golang.org
 
 # Which github repository and branch to use for testing with skopeo
 SKOPEO_REPO = containers/skopeo
-SKOPEO_BRANCH = master
+SKOPEO_BRANCH = release-1.1
```

Comment:
@vrothberg @mtrmac change Makefile like this?

Reply:
In… principle. Except that `release-1.1` uses c/image 5.5.

AFAICS there has never been a Skopeo release that uses 5.1; we went from 5.0.0 in 0.1.40 to 5.2.1 in 0.1.41.

And yet again, the latest bug refers to Podman 2.0.[45], which use c/image 5.5.1, not 5.1.x. Is 5.1 really the right target?

Reply:
should I commit to https://github.com/containers/image/tree/release-5.5?

Reply:
This PR is against `release-5.5`, so I think everything's in order?

Reply:
Apologies, I guess I caused the confusion in my comment in BZ where I asked to open against v5.1 but meant v5.5.

However, this PR is against v5.5, so we should be good :+1: